app/views/domains/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="box wide">
+  <h1><%= @domain.domain %></h1>
+  <% unless @domain.banned_at %>
```

Comment:
You need to remove this `if` or the form can't be used to unban domains.

Reply:
Now we can use the form for this. Check it please